internal/extsvc/types.go

```diff
@@ -70,3 +74,27 @@ type AccountID string
 // code host. It can be the string representation of an integer (e.g. GitLab and Bitbucket
 // Server) or a GraphQL ID (e.g. GitHub) depends on the code host type.
 type RepoID string
+
+// ParseConfig attempts to parse external service config into a strong typed
```

Comment:
```suggestion
// ParseConfig attempts to unmarshal the given JSON config into a configuration struct defined in the schema package.
```

I don't think we should say "strongly typed" when we return an `interface{}` 😄